yarn test fails with error `Cannot find module '@aws-sdk/util-dynamodb'`

### Steps to reproduce

Follow README.
Run `yarn test`

Output:
```console
Cannot find module '@aws-sdk/util-dynamodb' from 'node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/utils.js'

    Require stack:
      node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/utils.js
      node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/BatchExecuteStatementCommand.js
      node_modules/@aws-sdk/lib-dynamodb/dist/cjs/index.js
      src/getUserNames.spec.ts
```

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
     "@aws-sdk/lib-dynamodb": "3.21.0"
   },
   "devDependencies": {
+    "@aws-sdk/util-dynamodb": "^3.21.0",
     "@types/jest": "26.0.24",
     "aws-sdk-client-mock": "0.5.3",
     "jest": "27.0.6",

